crates/ironclaw_webui_v2/frontend/src/styles/app.css

```
@@ -0,0 +1,23 @@
+@import "tailwindcss";
+@import "../../../static/styles/app.css";
+
+@source "../../../static/js";
+
+@theme {
+  --font-sans: "Geist", "Inter", "SF Pro Display", "Helvetica Neue",
+    ui-sans-serif, system-ui, sans-serif;
+  --font-serif: "Newsreader", "Lyon Text", "Instrument Serif", Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)